site/en/docs/extensions/mv3/getstarted/index.md

```diff
@@ -137,146 +139,128 @@ extension uses a button to change the background color.
 ```
 
 Like the background script, this file needs to be designated as a popup in the manifest under
-[`page_action`][17].
+[`action`][17].
 
-```json
+```json/9-11
 {
   "name": "Getting Started Example",
-  "version": "1.0",
   "description": "Build an Extension!",
-  "permissions": ["storage"],
+  "version": "1.0",
+  "manifest_version": 3,
   "background": {
-    "scripts": ["background.js"],
-    "persistent": false
+    "service_worker": "background.js"
   },
-  "page_action": {
+  "permissions": ["storage"],
+  "action": {
     "default_popup": "popup.html"
-  },
-  "manifest_version": 3
+  }
 }
 ```
 
-Designation for toolbar icons is also included under `page_action` in the `default_icons` field.
+Our popup's HTLM references an external CSS file named `button.css`. Add another file to the
```

Comment:
Our popup -> This popup

(https://developers.google.com/style/person)

HTLM -> HTML

Reply:
Fixed in next commit